dissect/target/filesystems/tar.py

```diff
@@ -33,15 +33,20 @@ class TarFilesystem(Filesystem):
     def __init__(
         self,
         fh: BinaryIO,
-        base: Optional[str] = None,
-        tarinfo: Optional[tarfile.TarInfo] = None,
+        base: str | None = None,
```

Comment:
```suggestion
        base: str | None = None,
        *,
```
Maybe to make the arguments a bit more explicit.